Update signal.py

erh3cq opened the pull request:
Added an indexmin() and valuemin() similar to indexmax() and valuemax().
This file does not contain the new marker import feature. It kept raising errors on my build.

erh3cq committed:
Update signal.py

Added an indexmin() and valuemin() similar to  indexmax() and valuemax().
This file does not contain the new marker import feature.  It kept raising errors on my build.

erh3cq committed:
Merge pull request #1 from erh3cq/Update-signal

Update signal.py

francisco-dlp commented:
``valuemin`` is duplicated and ``indexmin`` is not present. Beside it requires adding unittests [like this one](https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/hyperspy/tests/signal/test_tools.py#L196).

erh3cq committed:
Update signal.py

added the indexmin, and deleted the duplicate.  How do I add a second file (test_tool.py) to this pull?

erh3cq committed:
Update test_tools.py

added test_indexmin() and test_valuemin()

francisco-dlp commented:
Looks good, thanks!
